native/tests/unit-test/src/core_event_bus.cpp

```diff
@@ -0,0 +1,212 @@
+#include "core/event/EventBus.h"
+#include "utils.h"
+
+DECLARE_EVENT_BUS(Test1)
+DECLARE_BUS_EVENT_ARG0(BusEvent0_Test1, Test1)
+DECLARE_BUS_EVENT_ARG1(BusEvent1_Test1, Test1, int)
+DECLARE_BUS_EVENT_ARG2(BusEvent2_Test1, Test1, int, const char *)
+DECLARE_BUS_EVENT_ARG3(BusEvent3_Test1, Test1, int, char *, float)
+
+DECLARE_EVENT_BUS(Test2)
+DECLARE_BUS_EVENT_ARG0(BusEvent0_Test2, Test2)
+DECLARE_BUS_EVENT_ARG1(BusEvent1_Test2, Test2, int)
+DECLARE_BUS_EVENT_ARG2(BusEvent2_Test2, Test2, int, const char *)
+DECLARE_BUS_EVENT_ARG3(BusEvent3_Test2, Test2, int, char *, float)
+
+struct Test1_ListenerT {
+    static int handleArg2_arg0;
+    static const char *handleArg2_arg1;
+    static int handleArg3_arg0;
+    static char *handleArg3_arg1;
+    static float handleArg3_arg2;
+    static void handleArg2(int arg0, const char *arg1) {
+        handleArg2_arg0 = arg0;
+        handleArg2_arg1 = arg1;
+    }
+    static void handleArg3(int arg0, char *arg1, float arg2) {
+        handleArg3_arg0 = arg0;
+        handleArg3_arg1 = arg1;
+        handleArg3_arg2 = arg2;
+    }
+};
+int Test1_ListenerT::handleArg2_arg0 = 0;
+const char *Test1_ListenerT::handleArg2_arg1 = nullptr;
+int Test1_ListenerT::handleArg3_arg0 = 0;
+char *Test1_ListenerT::handleArg3_arg1 = nullptr;
+float Test1_ListenerT::handleArg3_arg2 = 0;
+
+static int handleArg2_value1 = 0;
+static const char *handleArg2_value2 = nullptr;
+static void handleArg2(int arg0, const char *arg1) {
+    handleArg2_value1 = arg0;
+    handleArg2_value2 = arg1;
+}
```

Comment:
It is better to define them in anonymous namespace.

Reply:
fixed